src/providers/http_provider.py

```diff
@@ -93,7 +93,14 @@ def _get(
 
         for host in self.hosts:
             try:
-                return self._get_without_fallbacks(host, endpoint, path_params, query_params, stream)
+                data = self._get_without_fallbacks(host, endpoint, path_params, query_params, stream)
+                logger.debug({
+                    'msg': 'Get request',
+                    'url': endpoint,
+                    'data': f'{data}',
```

Comment:
Data from `GET` endpoints can be quite large. Validators data and keys info can be as large as `300mb` (or even larger). Are we sure we want to print it? @F4ever.

Reply:
Logging only a single request, as discussed